components/more-less/more-less.js

```diff
@@ -154,9 +159,16 @@ export class D2LMoreLess extends LocalizeMixin(LitElement)  {
 	}
 
 	render() {
+		var contentClasses = {
+			'more-less-content': true,
+			'more-less-transition': this.__transitionAdded
+		};
```

Comment:
Super-nit: we can start introducing ES6 things like `let`, `const`, etc. It's finally here!

Reply:
Were we unable to do this in Polymer 3?  Or we just didn't and should start?  My guess is the later...